### ISSUE TITLE
Squiz.WhiteSpace.MemberVarSpacing - inconsistent checking of member vars with comment

### ISSUE THREAD

**Issue by VasekPurchart** [squizlabs/PHP_CodeSniffer#414]
After upgrading to 2.x I have found this strange error, that was not reported in 1.5.x versions.

This code works fine obviously:

``` php
class Foo
{

    private $foo;

    private $bar;

}
```

but when you add an inline comment, it starts to throw an `Squiz.WhiteSpace.MemberVarSpacing.AfterComment` for the line with the comment:

``` php
class Foo
{

    private $foo; // comment

    private $bar;

}
```

This was not checked in previous versions, but is understandable - which one is correct behavior according to Squiz?

Strange is, that this situation is not reported consistently, because the following example produces no errors at all:

``` php
class Foo
{

    private $foo; // comment

    /**
     * @var type
     */
    private $bar;

}
```

**Comment by gsherwood** [comment]
That's because it is assuming that the comment is describing the following member var. It doesn't know that the comment is describing the member var on the same line.

So this is an error. It can assume that an inline comment like this is attached to the first member var and not the second.

**Comment by VasekPurchart** [comment]
Thank you for the fix, I've been testing master again and found out one even stranger situation in our codebase, which I missed last time:

``` php
class Foo
{

	/**
	 * @var integer
	 */
	private $foo; // comment

	private $bar;

}
```

meaning that there are both types of comments for the first member variable and an spacing error on the second one is detected. Again, I don't know what's the standpoint of Squiz on this (weird) situation, but I would probably expect the inline comment not to be taken into account at all (which would cover the simpler example from the OP as well).

**Comment by gsherwood** [comment]
I've fixed up this extra case. Thanks for testing.
